DataModels.py

```diff
@@ -73,13 +81,16 @@ def removeOndselWorkspaces(self):
 
         for i in range(len(self.workspaces) - 1, -1, -1):
```

Comment:
Same as above comment, to iterate list in reverse order use `reversed` method. Also, don't need to use `i`.

```python
for workspace in reversed(self.workspaces):
    ...
```